home/views/api/appuser.py

```diff
@@ -20,7 +20,7 @@
 
 # Determines whether Account is tester account, based on name prefix
 def is_tester(name_field: str) -> bool:
-    possible_prefixes = ["tester-", "tester ", "tester_"]
+    possible_prefixes = ["iwt-", "iwt ", "iwt_"]
```

Comment:
The inclusion of the second prefix `iwt` means the other two prefices are also included (redundant).

So we could just `possible_prefices = ["iwt"]`

Reply:
> The inclusion of the second prefix `iwt` means the other two prefices are also included (redundant).
> 
> So we could just `possible_prefices = ["iwt"]`

Yea, I noticed this while I was doing the quick search/replace that this just was. I'll change it since I agree with you - left it just to keep the change "atomic" FWIW, which is not much... better to remove useless code, most of the time...